iotwifi/wpacfg.go

```diff
@@ -114,17 +116,49 @@ rsn_pairwise=CCMP`
 			wpa.Log.Info("Hostapd DISABLED")
 			//cmd.Process.Kill()
 			//cmd.Wait()
-
-			return
-
+			return cmd
 		}
 		if strings.Contains(out, "uap0: AP-ENABLED") {
 			wpa.Log.Info("Hostapd ENABLED")
-			return
+			return cmd
 		}
 	}
 }
 
+// Status returns the AP status.
+func (wpa *WpaCfg) APStatus() (map[string]interface{}, error) {
+	cfgMap := make(map[string]interface{}, 0)
+
+	// get the standard stats
+	stateOut, err := exec.Command("hostapd_cli", "-i", "uap0", "status").Output()
+	if err != nil {
+		wpa.Log.Fatal("Got error checking state: %s", err.Error())
+		return cfgMap, err
+	}
+
+	// Remove the indexing associated with ssid, bssid, and bss
+	stateOut = bytes.ReplaceAll(stateOut, []byte("[0]"), []byte(""))
```

Comment:
hostapd can support multiple APs so some of the fields include an index.  Since we only use one we can strip them out for clarity.

Reply:
See comment below but since we're doing string operations converting to a string and then using the `strings` package seems appropriate.